server/Cargo.toml

```diff
@@ -4,5 +4,6 @@ cargo-features = ["resolver"]
 resolver = "2"
 
 members = [
-    "bastionai_app"
+    "bastionai_app",
+    "bastionai_common"
```

Comment:
What is the bastionai_common package for?

Reply:
Again, making things look similar to blindai.
Currently, it's used to hold the Networking struct.